src/databricks/labs/lsql/core.py

```diff
@@ -119,6 +120,10 @@ def __repr__(self):
         return f"Row({', '.join(f'{k}={v!r}' for (k, v) in zip(self.__columns__, self, strict=True))})"
 
 
+class DeltaConcurrentAppend(DatabricksError):
```

Comment:
Introduced this error

Reply:
`ConcurrentModification` to be precise - we can concurrently delete, append or update